CODEOWNERS

```diff
@@ -21,7 +21,7 @@
 /mysql/ @ruslanbes
 /node.js/ @pbachman
 /php/ @ruslanbes
-/python/ @tik9 @Amsal1
+/python/ @tik9 @Amsal1 @tsg405
```

Comment:
@TSG405 please also do not forget to add youself to the Readme.md inside column Responsible, thanks

Reply:
do not ignore my message please, if you want to be Codeowners you have to add your self here is well, thanks
![image](https://user-images.githubusercontent.com/13170022/194413456-eb8dc0a5-d4a1-4f88-b61f-5054e4ea9ca4.png)
and make sure you can answer qeustions.

Reply:
Okay, gotcha